test/functional/ui/screen.lua

```diff
@@ -54,12 +54,17 @@
 -- having the exact same set of attributes will be substituted by "{K:S}",
 -- where K is a key associated the attribute set via the second argument of
 -- "expect".
+-- If a transformation table is present, unexpected attribute sets in the final
+-- state is considered an error. To make testing simpler, a list of attribute sets that
+-- should be ignored can be passed as a third argument. Alternatively, this third
+-- argument can be "true" to indicate that all unexpected attribute sets should be ignored.
```

Comment:
Do we also use a maximum of 80 columns for lua files? `:set colorcolumn=80` shows some
lines longer than that.